### ISSUE TITLE
Test release 26.0

### ISSUE THREAD

**Issue by guerler** [galaxyproject/iwc#1092]

**Comment by mvdbeek** [comment]
# Test Results (powered by Planemo)


## Test Summary







<div class="progress">
  <div class="progress-bar progress-bar-success" style="width: 92.7927927927928%" aria-valuenow="103" aria-valuemin="0" aria-valuemax="111" data-toggle="tooltip" title="103 Passed">
  </div>
  <div class="progress-bar progress-bar-warning" style="width: 0.0%" aria-valuenow="0" aria-valuemin="0" aria-valuemax="111" data-toggle="tooltip" title="0 Skipped">
  </div>
  <div class="progress-bar progress-bar-danger" style="width: 7.207207207207207%" aria-valuenow="8" aria-valuemin="0" aria-valuemax="111" title="8 Failed or Errored">
  </div>
</div>


| Test State | Count |
| ---------- | ----- |
| Total      | 111 |
| Passed     | 103 |
| Error      | 3 |
| Failure    | 5 |
| Skipped    | 0 |



<details open><summary>Errored Tests</summary>















* <summary class="light-red">&#10060; Assembly-decontamination-VGP9.ga_1</summary>



























































* <summary class="light-red">&#10060; RepeatMasking-Workflow.ga_0</summary>























































* <summary class="light-red">&#10060; fragment-based-docking-scoring.ga_0</summary>








































































































</details>


<details open><summary>Failed Tests</summary>





























* <summary class="light-red">&#10060; Generic-variation-analysis-on-WGS-PE-data.ga_0</summary>

















* <summary class="light-red">&#10060; Mass_spectrometry__LC-MS_preprocessing_with_XCMS.ga_0</summary>





















































































* <summary class="light-red">&#10060; goseq-go-kegg-enrichment-analsis.ga_0</summary>





























* <summary class="light-red">&#10060; kmer-profiling-hifi-trio-VGP2.ga_0</summary>











































* <summary class="light-red">&#10060; pe-wgs-variation.ga_0</summary>




































</details>


<details ><summary>Passed Tests</summary>



* <summary class="light-green">&#9989; Allele-based-Pathogen-Identification.ga_0</summary>





* <summary class="light-green">&#9989; Assembly-Hifi-HiC-phasing-VGP4.ga_0</summary>





* <summary class="light-green">&#9989; Assembly-Hifi-HiC-phasing-VGP4.ga_1</summary>





* <summary class="light-green">&#9989; Assembly-Hifi-Trio-phasing-VGP5.ga_0</summary>





* <summary class="light-green">&#9989; Assembly-Hifi-only-VGP3.ga_0</summary>





* <summary class="light-green">&#9989; Assembly-decontamination-VGP9.ga_0</summary>







* <summary class="light-green">&#9989; Assembly-polishing-with-long-reads.ga_0</summary>





* <summary class="light-green">&#9989; BREW3R.ga_0</summary>





* <summary class="light-green">&#9989; Functional_annotation_of_protein_sequences.ga_0</summary>





* <summary class="light-green">&#9989; Galaxy-Workflow-lncRNAs_annotation_workflow.ga_0</summary>





* <summary class="light-green">&#9989; Gene-based-Pathogen-Identification.ga_0</summary>





* <summary class="light-green">&#9989; Generate-Nx-and-Size-plots-for-multiple-assemblies.ga_0</summary>







* <summary class="light-green">&#9989; Generic-variation-analysis-reporting.ga_0</summary>





* <summary class="light-green">&#9989; Genome-assembly-with-Flye.ga_0</summary>





* <summary class="light-green">&#9989; Genome_annotation_with_braker3.ga_0</summary>





* <summary class="light-green">&#9989; Genome_annotation_with_maker_short.ga_0</summary>





* <summary class="light-green">&#9989; MAGs-generation.ga_0</summary>





* <summary class="light-green">&#9989; Mass-spectrometry__GCMS-with-metaMS.ga_0</summary>







* <summary class="light-green">&#9989; Mitogenome-Assembly-VGP0.ga_0</summary>





* <summary class="light-green">&#9989; Preprocessing-and-Clustering-of-single-cell-RNA-seq-data-with-Scanpy.ga_0</summary>





* <summary class="light-green">&#9989; Purge-duplicate-contigs-VGP6.ga_0</summary>





* <summary class="light-green">&#9989; Purging-duplicates-one-haplotype-VGP6b.ga_0</summary>





* <summary class="light-green">&#9989; QCxMS-Spectra-Prediction-from-SDF.ga_0</summary>





* <summary class="light-green">&#9989; QIIME2-III-V-Phylogeny-Rarefaction-Taxonomic-Analysis.ga_0</summary>





* <summary class="light-green">&#9989; QIIME2-IIa-denoising-and-feature-table-creation-single-end.ga_0</summary>





* <summary class="light-green">&#9989; QIIME2-IIb-denoising-and-feature-table-creation-paired-end.ga_0</summary>





* <summary class="light-green">&#9989; QIIME2-Ia-multiplexed-data-single-end.ga_0</summary>





* <summary class="light-green">&#9989; QIIME2-Ib-multiplexed-data-paired-end.ga_0</summary>





* <summary class="light-green">&#9989; QIIME2-Ic-demultiplexed-data-single-end.ga_0</summary>





* <summary class="light-green">&#9989; QIIME2-Id-demultiplexed-data-paired-end.ga_0</summary>





* <summary class="light-green">&#9989; QIIME2-VI-diversity-metrics-and-estimations.ga_0</summary>







* <summary class="light-green">&#9989; Scaffolding-BioNano-VGP7.ga_0</summary>





* <summary class="light-green">&#9989; Scaffolding-HiC-VGP8.ga_0</summary>





* <summary class="light-green">&#9989; Velocyto-on10X-filtered-barcodes.ga_0</summary>





* <summary class="light-green">&#9989; Velocyto-on10X-from-bundled.ga_0</summary>





* <summary class="light-green">&#9989; WF5_Data_Interpretation_Worklow.ga_0</summary>





* <summary class="light-green">&#9989; WGS-PE-variant-calling-in-haploid-system.ga_0</summary>





* <summary class="light-green">&#9989; amr_gene_detection.ga_0</summary>





* <summary class="light-green">&#9989; atacseq.ga_0</summary>





* <summary class="light-green">&#9989; average-bigwig-between-replicates.ga_0</summary>





* <summary class="light-green">&#9989; bacterial_genome_annotation.ga_0</summary>





* <summary class="light-green">&#9989; bacterial_genome_assembly.ga_0</summary>





* <summary class="light-green">&#9989; bacterial_quality_and_contamination_control_post_assembly.ga_0</summary>





* <summary class="light-green">&#9989; baredSC-1d-logNorm.ga_0</summary>





* <summary class="light-green">&#9989; baredSC-2d-logNorm.ga_0</summary>





* <summary class="light-green">&#9989; cgmlst_bacterial_genome.ga_0</summary>





* <summary class="light-green">&#9989; chic-fastq-to-cool-hicup-cooler.ga_0</summary>





* <summary class="light-green">&#9989; chipseq-pe.ga_0</summary>





* <summary class="light-green">&#9989; chipseq-sr.ga_0</summary>





* <summary class="light-green">&#9989; consensus-from-variation.ga_0</summary>





* <summary class="light-green">&#9989; consensus-peaks-atac-cutandrun.ga_0</summary>





* <summary class="light-green">&#9989; consensus-peaks-chip-pe.ga_0</summary>





* <summary class="light-green">&#9989; consensus-peaks-chip-sr.ga_0</summary>





* <summary class="light-green">&#9989; cutandrun.ga_0</summary>





* <summary class="light-green">&#9989; cutandrun.ga_1</summary>





* <summary class="light-green">&#9989; dada2_paired.ga_0</summary>









* <summary class="light-green">&#9989; gromacs-dctmd.ga_0</summary>





* <summary class="light-green">&#9989; gromacs-mmgbsa.ga_0</summary>





* <summary class="light-green">&#9989; hi-c-map-for-assembly-manual-curation.ga_0</summary>





* <summary class="light-green">&#9989; hi-c-map-for-assembly-manual-curation.ga_1</summary>





* <summary class="light-green">&#9989; hic-fastq-to-cool-hicup-cooler.ga_0</summary>





* <summary class="light-green">&#9989; hic-fastq-to-pairs-hicup.ga_0</summary>





* <summary class="light-green">&#9989; hic-juicermediumtabix-to-cool-cooler.ga_0</summary>





* <summary class="light-green">&#9989; host-or-contamination-removal-on-long-reads.ga_0</summary>





* <summary class="light-green">&#9989; host-or-contamination-removal-on-short-reads.ga_0</summary>





* <summary class="light-green">&#9989; influenza-consensus-and-subtyping.ga_0</summary>





* <summary class="light-green">&#9989; iwc-clinicalmp-database-generation.ga_0</summary>





* <summary class="light-green">&#9989; kmer-profiling-hifi-VGP1.ga_0</summary>







* <summary class="light-green">&#9989; mapseq-to-ampvis2.ga_0</summary>





* <summary class="light-green">&#9989; metagenomic-genes-catalogue.ga_0</summary>





* <summary class="light-green">&#9989; metagenomic-raw-reads-amr-analysis.ga_0</summary>





* <summary class="light-green">&#9989; metaprosip.ga_0</summary>





* <summary class="light-green">&#9989; mfassignr.ga_0</summary>





* <summary class="light-green">&#9989; mgnify-amplicon-pipeline-v5-complete.ga_0</summary>





* <summary class="light-green">&#9989; mgnify-amplicon-pipeline-v5-its.ga_0</summary>





* <summary class="light-green">&#9989; mgnify-amplicon-pipeline-v5-quality-control-paired-end.ga_0</summary>





* <summary class="light-green">&#9989; mgnify-amplicon-pipeline-v5-quality-control-single-end.ga_0</summary>





* <summary class="light-green">&#9989; mgnify-amplicon-pipeline-v5-rrna-prediction.ga_0</summary>





* <summary class="light-green">&#9989; mgnify-amplicon-summary-tables.ga_0</summary>





* <summary class="light-green">&#9989; multiplex-tma.ga_0</summary>





* <summary class="light-green">&#9989; ont-artic-variation.ga_0</summary>





* <summary class="light-green">&#9989; parallel-accession-download.ga_0</summary>





* <summary class="light-green">&#9989; parallel-accession-download.ga_1</summary>





* <summary class="light-green">&#9989; pe-artic-variation.ga_0</summary>





* <summary class="light-green">&#9989; pe-illumina-simple-virus-calling-and-consensus.ga_0</summary>





* <summary class="light-green">&#9989; pe-illumina-simple-virus-calling-and-consensus.ga_1</summary>





* <summary class="light-green">&#9989; pe-wgs-ivar-analysis.ga_0</summary>







* <summary class="light-green">&#9989; pox-virus-half-genome.ga_0</summary>





* <summary class="light-green">&#9989; protein-ligand-complex-parameterization.ga_0</summary>





* <summary class="light-green">&#9989; pseudo-bulk_edgeR.ga_0</summary>





* <summary class="light-green">&#9989; quality_and_contamination_control_raw_reads.ga_0</summary>





* <summary class="light-green">&#9989; rnaseq-de-filtering-plotting.ga_0</summary>





* <summary class="light-green">&#9989; rnaseq-pe.ga_0</summary>





* <summary class="light-green">&#9989; rnaseq-sr.ga_0</summary>





* <summary class="light-green">&#9989; scrna-seq-fastq-to-matrix-10x-cellplex.ga_0</summary>





* <summary class="light-green">&#9989; scrna-seq-fastq-to-matrix-10x-v3.ga_0</summary>





* <summary class="light-green">&#9989; se-wgs-variation.ga_0</summary>





* <summary class="light-green">&#9989; segmentation-and-counting.ga_0</summary>





* <summary class="light-green">&#9989; short-read-quality-control-and-trimming.ga_0</summary>





* <summary class="light-green">&#9989; sra-manifest-to-concatenated-fastqs.ga_0</summary>





* <summary class="light-green">&#9989; taxonomic-rank-abundance-summary-table.ga_0</summary>





* <summary class="light-green">&#9989; tissue-micro-array-analysis.ga_0</summary>





* <summary class="light-green">&#9989; variation-reporting.ga_0</summary>




</details>
